tests/test_order.py

```diff
@@ -8,13 +8,22 @@
 from unittest.mock import patch
 from pyetrade import order
 from collections import OrderedDict
+from nose.tools import *

```

Comment:
Is there a need for nose? Unit test package has: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual
This would remove the need for the requirement. I have been slowly migrating pyetrade away from `unittest` over to `pytest`. 
If there is a need to keep the `nose` import please do not use a wildcard import.

Reply:
ok fixed using TestCase.assertEqual.